docs/src/documentation.md

```diff
@@ -30,3 +30,14 @@ whenever someone pushes to the develop branch on the GitHub server. The updated
 documentation can then be accessed [here](https://joachimbrand.github.io/Rimu.jl/dev/).
 
 Previews for pull-requests can be accessed by replacing 101 in the following link with the PR number: [https://joachimbrand.github.io/Rimu.jl/previews/PR101/](https://joachimbrand.github.io/Rimu.jl/previews/PR101/)
+
+### Example scripts
+
+Examples should be added to the `scripts` folder, in the form of `.jl` files suitable for 
+parsing by [`Literate`](https://github.com/fredrikekre/Literate.jl). The process of generating 
+documentation is automated in the `docs/make.jl` file and assumes that the following line is 
+at (or near) the top of the script:
+```
+# # Example N: Title
+```
+where the number `N` and `Title` will be extracted automatically.
```

Comment:
Thanks, this is good!

I'm thinking that we should add instructions about adding a test for each new example script in `test/examples.jl` as currently this has to be done manually. Alternatively one could think of automating the operations of `test/examples.jl` such that a new example script would be picked up automatically (and, e.g. just check that it runs without errors).